internal/ee/service/file_download_ssrf_test.go

```
@@ -0,0 +1,79 @@
+	defer server.Close()
+
+	loopbackTask := &task.Task{FileURL: server.URL + "/features.csv"}
+
+	t.Run("streaming processor", func(t *testing.T) {
+		serverHit.Store(false)
+		sp := NewStreamingProcessor(httpclient.NewDefaultClient(), log)
+
+		start := time.Now()
+		body, err := sp.downloadFileStream(context.Background(), loopbackTask)
+		if err == nil {
+			body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `body.Close` is not checked

(errcheck)